x-pack/plugin/snapshot-based-recoveries/src/internalClusterTest/java/org/elasticsearch/xpack/snapshotbasedrecoveries/recovery/SnapshotBasedIndexRecoveryIT.java

```diff
@@ -1255,7 +1256,10 @@ public void testRecoveryRetryKeepsTheGrantedSnapshotFileDownloadPermit() throws
         );
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/90665")
+    @TestLogging(
```

Comment:
Is this just in case that another error re-appears and we'd like to check the trace log?

Reply:
Yes, I'm 99% sure that this should fix the issue, but in case the error happens again we would have more information to fix it.

Reply:
I wonder if it will produce much more unnecessary output that we'll needlessly keep. Do you think it'd be a good idea to make a issue/reminder to remove this after a couple of months of no problems?